app/views/issues/_attributes.rhtml

```diff
@@ -883,8 +883,78 @@ function toUtcDate(date) {
   return date && new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate(), date.getUTCHours(), date.getUTCMinutes(), date.getUTCSeconds())
 }
 
+function getTrackingUrls(values) {
+  let regexObjList = 
+    [
+      {
+        "regex": [
+          /^1\s*Z\s*(?:[A-Z0-9]\s*){6,6}(?:[A-Z0-9]\s*){2,2}(?:[A-Z0-9]\s*){7,7}[0-9]/g
+        ],
+        "trackingUrl": "https://wwwapps.ups.com/WebTracking/track?track=yes&trackNums=%" 
+      },
+      {
+        "regex": [
+          /^([0-9]\s*){11}[0-9]/g,
+          /^([0-9]\s*){14}[0-9]/g
+        ],
+        "trackingUrl": "https://www.fedex.com/fedextrack/?tracknumbers=%"
+      },
+      {
+        "regex": [
+          /^([0-9]\s*){2}([0-9]\s*){9}([0-9]\s*){8}[0-9]/g,
+          /^((?:4\s*2\s*0\s*([0-9]\s*){5})?9\s*[12345]\s*?([0-9]\s*){2}([0-9]\s*){2}([0-9]\s*){8}([0-9]\s*){11}|([0-9]\s*){7}[0-9])/g
+        ],
+        "trackingUrl": "https://tools.usps.com/go/TrackConfirmAction.action?tLabels=%"
+      }
+    ]
+
+  var trackingUrls = [];
+  values.forEach(v => {
+    var vUpperCase = v.toLocaleUpperCase()
+    var foundRegex = regexObjList.find(regexObj => {
+      return regexObj["regex"].some(regex => {
+        return !!vUpperCase.match(regex)
+      })
+    }) || {}
+
+    var trackingUrl = foundRegex["trackingUrl"]
+
+    if (!!trackingUrl) trackingUrls.push([v, trackingUrl.replace("%", vUpperCase)])
+    else if (v.length != 0) trackingUrls.push([v, `https://www.google.com/search?q=${vUpperCase}+tracking+number`])
+    else trackingUrls
+  })
+
+  return trackingUrls;
+}
+
+function setTrackingUrls() {
+  let trackingNumberLabelTag = $$("th").find(tag => tag.outerText.toLocaleLowerCase().indexOf("tracking number") != -1)
+  let trackingNumberValueTag = trackingNumberLabelTag.nextElementSibling
+  let cleanTrackingNumberValue = trackingNumberValueTag.textContent.replace(" ", "")
```

Comment:
```suggestion
  let cleanTrackingNumberValue = trackingNumberValueTag.textContent.replace(/\s+/g, "")
```
If the goal is to remove whitespace anywhere in the string, this would work better in case there are multiple spaces. Otherwise, if you're just looking to remove whitespace at beginning and end, `.trim()` would work.